.github/ISSUE_TEMPLATE/IMPROVEMENT-REQUEST.yml

```
@@ -0,0 +1,39 @@
+name: "💡 Improvement Request"
+description: Create a new ticket for a new improvement request
```

```suggestion
name: "Improvement Request"
description: Create a ticket to request an improvement of something that already exists.
```
No emojis please and changed the description.